build/docker/Makefile

```diff
@@ -7,7 +7,7 @@ REPO=redisearch
 REDIS_VERSION=7.2-rc1
 OSNICK.official=bullseye
 
-INT_BRANCHES=2.6 2.4 2.2 2.0 1.6
+INT_BRANCHES=2.8 2.6 2.4 2.2 2.0 1.6
```

Comment:
Why is this change part of this documentation PR?

Reply:
Maybe something automatic Omer, I didn't touch this part